homeassistant/components/nam/sensor.py

```diff
@@ -92,3 +97,17 @@ def available(self) -> bool:
         return available and bool(
             getattr(self.coordinator.data, self.sensor_type, None)
         )
+
+
+class NAMSensorUptime(NAMSensor):
+    """Define an Nettigo Air Monitor uptime sensor."""
+
```

Comment:
We usually recommend setting `entity_registry_enabled_default` to `True` for uptime and wifi seconds since they generate a lot of state changes and can fill up a database quickly.

Reply:
Do you mean `False`? This sensor is disabled by default, look [here](https://github.com/home-assistant/core/blob/b5739ecb713e9aff89c7b23f650ff9806ba4790a/homeassistant/components/nam/const.py#L136).

Reply:
👍